app/(app)/scan/scan-result.tsx

```diff
@@ -3,12 +3,36 @@ import { router, useLocalSearchParams } from 'expo-router'
 import ContinueButton from '@/components/navigation/ContinueButton'
 import ContentWithAvatar from '@/components/screen-layout/ContentWithAvatar'
 import ScreenViewCentered from '@/components/screen-layout/ScreenViewCentered'
+import { useSnackbar } from '@/components/screen-layout/Snackbar/useSnackbar'
 import { ScanResultEnum } from '@/modules/backend/openapi-generated'
 
 type ScanResultSearchParams = {
   scanResult: ScanResultEnum
 }
 
+const scanResultTextsMap: Record<ScanResultEnum, Record<string, string>> = {
+  [ScanResultEnum.NoViolation]: {
+    title: 'Bez priestupku',
```

Comment:
I'd seriously start consideringadding i18n as soon as possible as the number of texts is rapidly increasing

Reply:
it's on my agenda after next week's presentations

Reply:
cc @radoslavzeman as we discussed this offline